source/MaterialXGenShader/ShaderNode.h

```diff
@@ -21,6 +21,9 @@ using ShaderOutputPtr = shared_ptr<class ShaderOutput>;
 using ShaderNodePtr = shared_ptr<class ShaderNode>;
 using ShaderInputSet = std::set<ShaderInput*>;
 
+// List of possible Flags
+static unsigned int NOT_RENAMABLE_FLAG = 1 << 0;
```

Comment:
As there is a "name" member maybe something with the word variable in it instead or this
flag makes it sound like you shouldn't change the name which you can. VARIABLE_NOT_RENAMABLE ?